logfire/_internal/integrations/starlette.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+from opentelemetry.instrumentation.starlette import StarletteInstrumentor
+from starlette.applications import Starlette
+
+
+def instrument_starlette(app: Starlette, **kwargs: Any):
```

Comment:
As a user, I would have to go to the OTel documentation to see which ones are the kwargs. I think we should correctly type hint instead of using the kwargs.

If the argument for doing it like this is because they may break our API, we can just do the `if TYPE_CHECKING` trick. It just provides a far better experience.

Reply:
Yes, we should do that eventually.